bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/wizards/datatransfer/FileSystemExportOperation.java

```diff
@@ -227,7 +228,9 @@ protected void exportFile(IFile file, IPath location)
 						.queryOverwrite(properPathString);
 
 				if (overwriteAnswer.equals(IOverwriteQuery.CANCEL)) {
-					throw new InterruptedException();
+					InterruptedException interruptedException = new InterruptedException();
+					interruptedException.initCause(new OperationCanceledException());
+					throw interruptedException;
```

Comment:
Same as above: Cancel may not only occur upon user cancellation. All consumers of the exception (`WizardFileSystemResourceExportPage1`) do not use the exception contents.